lighthouse-core/gather/gatherers/http-redirect.js

```diff
@@ -36,39 +36,20 @@ class HTTPRedirect extends Gatherer {
   }
 
   afterPass(options) {
-    // Reset the options.
-    options.url = this._preRedirectURL;
-
-    // Allow override for faster testing.
-    const timeout = options._testTimeout || 10000;
-
-    const securityPromise = options.driver.getSecurityState()
-      .then(state => {
-        return {
-          value: state.schemeIsCryptographic
-        };
+    let isRedirect = false;
+
+    if(options.driver._httpsArr.length !== 0) {
```

Comment:
I'm not really understanding why this is necessary @ev1stensberg could you elaborate? As @brendankenny mentioned we're already saving every network request that is issued (which will have flags for redirect, status code, scheme, etc) that you could check in `options`. I'm also interested why jakearchibald's site did not work with the previous approach, the `afterPass` is already quite some time after load and the page has settled, maybe we should just fix the redirect monitor instead if we're not picking up those changes :)

UPDATE: oh wait, does he do a `location = ...` too? we probably only examine redirects right now